docker/Dockerfile

```diff
@@ -17,7 +31,7 @@ RUN apk add --no-cache python3 make g++
 
 WORKDIR /kjudge/frontend
 
-COPY ./frontend/package.json ./frontend/yarn.lock .
+COPY ./frontend/package.json ./frontend/yarn.lock ./
```

Comment:
Best practices seems to [not do this](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy).

Reply:
I'm not sure which line of the docs you're referring to. On the contrary, the dockerfile-utils linter tells me that I shouldn't be doing this (the error was dockerfile-utils(23)). Not sure what it was trying to say, but I thought might as well add the slash to stop it from complaining.